contrib/platform/test/com/sun/jna/platform/win32/EnvironmentVarsTest.java

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2007 Timothy Wall, All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+package com.sun.jna.platform.win32;
+
+import java.util.Map;
+import java.util.Random;
+
+import org.junit.Test;
+
+import com.sun.jna.Native;
+
+/**
+ * @author lgoldstein
+ */
+public class EnvironmentVarsTest extends AbstractWin32TestSupport {
```

Comment:
_Kernel32Util_EnvironmentFunctionsTests_ is a bad name for a Java class - also see my previous remark about grouping tests that cover a specific API. As with _Kernel32_, _Kernel32Util_ is too generic to have a single test class cover all its APIs.